### DISCUSSION TITLE
black screen when viewing in Sweden

### DISCUSSION THREAD

**Discussion by kpetrow** [Ylianst/MeshCentral#5808] [Issues]
We have some clients that are in Sweden and can not view any mesh agent screen through the webviewer. We tested in various other countries and have not seen the issue.  Is there any specific reason why this would happen in Sweden? chrome debugger is not showing any errors. 

When did test rtc connection using https://test.8x8.vc/ and see a "Reflexive connectivity"  failure but am not sure how to resolve or if it makes a difference. Is there any other debugging that we can do to resolve this?  the issue is being seen on different networks/routers, but our ec2 instance in Stockholm is not seeing the issue.  Any suggestions?

**Comment by kpetrow** [comment]
This has to do with Mesh agent sending translated files to sweden.  I made some custom changes to sharing.handlebars to hide the toolbars and autoclick the connect button.  Where would i be able to alter sharing.handlebars for all translations?

**Comment by kpetrow** [comment]
I added to the translation_sv file. Is there a better way to inject JS into sharing.handlebars page?

**Comment by dinger1986** [comment]
@kpetrow did you sort this out?
